proposed/2021/VulnerabilityInfoInPMUI.md

```diff
@@ -0,0 +1,52 @@
+# Expose package vulnerability information in the Package Manager UI - Technical Spec
+
+* Status: **In Review**
+* Author(s): [Jean-Pierre Briedé](https://github.com/jebriede)
+* Issue: [10981](https://github.com/NuGet/Home/issues/10981) Expose package vulnerabilities information in the PMUI
+* Type: Feature
+
+## Problem Background
+
+Developers that install NuGet packages through Visual Studio may be installing packages that have known vulnerabilities or they may have a project that is depending on a vulnerable version of a NuGet package. There is currently no information in the Package Manager UI that indicates that a package version is vulnerable so developers may continue to use a vulnerable package in their project. Users should be able to see a warning about a package version they are browsing or one they have installed so they can be informed of the vulnerability and hopefully update to a different version of the package that is not vulnerable or use a different package.
```

Comment:
> Users should be able to see a warning about a package version they are browsing or one they have installed so they can be informed of the vulnerability.

Is this going to the case even for vulnerable transitive dependent packages also? Let's say I am searching for `abcd` package from a source. This package and direct dependencies don't have any known vulnerability, but a transitive dependency has a vulnerability.  Curious about what the user experience is going to be in this case.

Looking at this https://github.com/NuGet/Home/blob/dev/proposed/2020/PackageVulnerability/FlagVulnerablePackages.md#display-vulnerabilities-in-visual-studio-pmui doc, I think it covers transitive dependencies also but not referenced in this spec. Hence not sure.

Reply:
Transitive dependencies is a separate feature and exposing vulnerabilities on them will be another iteration on vulnerabilities when transitive dependencies is built.